cypress/plugins/github.js

```diff
@@ -0,0 +1,193 @@
+const Octokit = require('@octokit/rest');
```

Comment:
These tests are fantastic. I'd rather see them run against a mock server implementation than GitHub's API, maybe we can swap that in while still using the existing tests? I remember reading about Apollo helping with testability but haven't looked into it much.

Reply:
Let me search for a bit, maybe we can set up a local GraphQL server or something

Reply:
If it's disabled I'd be cool with getting this merged and revisiting later.